otherlibs/stdune/ansi_color.mli

```diff
@@ -1,84 +1,46 @@
 module Style : sig
-  type t
+  type t =
+    | Fg_default
+    | Fg_black
+    | Fg_red
+    | Fg_green
+    | Fg_yellow
+    | Fg_blue
+    | Fg_magenta
+    | Fg_cyan
+    | Fg_white
+    | Fg_bright_black
+    | Fg_bright_red
+    | Fg_bright_green
+    | Fg_bright_yellow
+    | Fg_bright_blue
+    | Fg_bright_magenta
+    | Fg_bright_cyan
+    | Fg_bright_white
+    | Bg_default
+    | Bg_black
+    | Bg_red
+    | Bg_green
+    | Bg_yellow
+    | Bg_blue
+    | Bg_magenta
+    | Bg_cyan
+    | Bg_white
+    | Bg_bright_black
+    | Bg_bright_red
+    | Bg_bright_green
+    | Bg_bright_yellow
+    | Bg_bright_blue
+    | Bg_bright_magenta
+    | Bg_bright_cyan
+    | Bg_bright_white
+    | Bold
+    | Dim
+    | Italic
+    | Underlined
```

Comment:
Unrelated, but can you rename to `Underline`? It seems weird to keep only one style in the past tense.